presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3000,6 +3000,28 @@ public void testOrderByWithSimilarExpressions()
                 "SELECT 1, 172799.49, 1");
     }
 
+    @Test
+    public void testComplexOrderBy()
+    {
+        assertQuery("SELECT\n" +
+                "  nationkey\n" +
+                ", stories_total_roas + stories_total_roas\n" +
```

Comment:
In the rest of this file we don't use new lines (`\n`) and don't place commas on the next line. Why here?

Reply:
Actually we use `\n`. Take a look at: `com.facebook.presto.tests.AbstractTestQueries#testRightJoinWithNullValues` for instance.

Reply:
@sopel39 Oh... I see. Some queries have `\n` some don't. I assume the ones who do were copy-pasted from somewhere and the ones that don't were written from scratch. I find the presence of `\n` making it harder to read. I had to reconstruct the query manually in a text editor to understand what it is doing. Would you consider rewriting it into fewer lines and replacing `stories_total_roas` with something simpler?

```
select nationkey, value + value
from (
  select nationkey, COALESCE(value, 0) value
  from (
    select nationkey, 1 as value from nation order by 1 asc
  )
)
```

Reply:
Removed `\n`